packages/devtools_app/lib/src/performance/performance_screen.dart

```diff
@@ -380,244 +364,70 @@ class _SecondaryControls extends StatelessWidget {
-                  ),
-                  TextSpan(
-                    text: '.',
-                    style: theme.subtleTextStyle,
-                  ),
-                ],
-              ),
+            TextSpan(
+              text: 'frame times may be negatively affected',
+              style: textStyle.copyWith(color: rasterJankColor),
+            ),
+            TextSpan(
+              text: '.',
+              style: textStyle,
             ),
-            const SizedBox(height: denseSpacing),
-            // TODO(kenz): link to documentation for each of these features when
-            // docs are available.
-            for (final serviceExtension
-                in EnhanceTracingOverlay.tracingServiceExtensions)
-              ServiceExtensionCheckbox(service: serviceExtension),
           ],
         ),
       ),
     );
   }
 }
 
-class DebuggingOptionsDialog extends StatelessWidget {
+class MoreDebuggingOptionsButton extends StatelessWidget {
+  const MoreDebuggingOptionsButton({Key key}) : super(key: key);
+
+  static const _width = 625.0;
+
   @override
   Widget build(BuildContext context) {
-    final theme = Theme.of(context);
-    return DevToolsDialog(
-      title: dialogTitleText(theme, 'Debugging Options'),
-      includeDivider: false,
-      content: Container(
-        width: defaultDialogWidth,
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Text(
-              'When toggling on/off a rendering layer, you will need '
-              'to reproduce activity in your app to see the effects of the '
-              'debugging option.',
-              style: theme.subtleTextStyle,
-            ),
-            const SizedBox(height: defaultSpacing),
-            ServiceExtensionCheckbox(service: disableClipLayers),
-            ServiceExtensionCheckbox(service: disableOpacityLayers),
-            ServiceExtensionCheckbox(service: disablePhysicalShapeLayers),
-          ],
-        ),
-      ),
-      actions: [
-        DialogCloseButton(),
+    return ServiceExtensionCheckboxOverlayButton(
+      title: 'More debugging options',
+      icon: Icons.build,
+      tooltip: 'Opens a list of options you can use to help debug performance',
+      minScreenWidthForTextBeforeScaling:
+          _SecondaryControls.minScreenWidthForTextBeforeScaling,
+      extensions: [
+        disableClipLayers,
+        disableOpacityLayers,
+        disablePhysicalShapeLayers,
       ],
+      overlayDescription: Text(
+        'When toggling on/off a rendering layer, you will need '
+        'to reproduce activity in your app to see the effects of the '
+        'debugging option. All layers are rendered by default - disabling a '
+        'layer may help you identify expensive operations in your app.',
+        style: Theme.of(context).subtleTextStyle,
+      ),
+      overlayWidth: _width,
```

Comment:
why is this width not scaled by the font scale factor?

Reply:
done